Removing portal groups and adding dco Team

mrpatrickwest opened the pull request:
Biggest modification is to the person code where I replaced
associatedDCOPortalGroup with the faux property for member of making
sure that the role is MemberRole and the org is a dco:Team. The other
changes are just changing associatedDCOPortalGroup to associatedDCOTeam.

Commit:
Removing portal groups and adding dco Team

Biggest modification is to the person code where I replaced
associatedDCOPortalGroup with the faux property for member of making
sure that the role is MemberRole and the org is a dco:Team. The other
changes are just changing associatedDCOPortalGroup to associatedDCOTeam.

Commit:
Changing from portal group to team

Added team facet to publications. Consistency in default es and script
and links in html. Problem with publication year being a long when
passed a string

Commit:
Added span7 to the person-info section

The text was being pushed down below the image. The div should have been
using both the person-info class and the span7 class.

Commit:
Filtering out team members not on team anymore

mrpatrickwest commented:
This is all working in production. Still need to work on communities before merging the pull request.

Commit:
Updates to get working in production

Commit:
datasets html matches production

zednis commented on ingest/mappings/publication.json:
```diff
@@ -167,7 +188,7 @@
       },
       "publicationYear":{
-        "type":"long"
+        "type":"string"
```
Why change publication year to string?  Have we confirmed sorting still works as expected?

zednis commented on ingest/ingestHelpers.py:
```diff
@@ -162,9 +162,9 @@ def main(get_objects_query_location, describe_object_query_location,
-        bulk_str = '\n'.join(ingestSomething.records) + '\n'
-        ingestSomething.publish(bulk=bulk_str, endpoint=args.es, rebuild=args.rebuild)
\ No newline at end of file
+        bulk_str = '\n'.join(ingestSomething.records) + '\n\n'
```
Why the double newline?

Commit:
Fixed datatype mapping title discrepancy

The index was title but the field was called name. Changed name to title
and it all seems to work now

mrpatrickwest commented:
Remember we had to add a newline to the end of the bulk file. In this case, though, adding the newline didn't work, but adding two newlines works. Very odd. Adding one newline works everywhere else, just not in datasets.

mrpatrickwest commented:
tested out the rest of the ingest scripts on dco.tw and they all seem to be working

zednis commented:
:shipit:
